.github/workflows/publish.yml

```diff
@@ -44,8 +44,8 @@ jobs:
 
     - name: Merge back to develop
       run: |
-        git fetch --depth 2 origin develop
+        git fetch --depth 10 origin develop
         git checkout develop
-        git fetch --shallow-since="$(git show -s --format=%ci develop~1)" origin master develop
+        git fetch --shallow-since="$(git show -s --format=%ci develop~2)" origin master develop
```

Comment:
Why only `2` here? (`develop~2`). Ok for me, but not sure if this is the behavior we want.

Reply:
This depth is not the same as `--depth` on the fetch above. If I understand this correctly, this will go back one merge request at the time. But I'm really unsure about this. As far as I can tell, the amount of commits we fetch above must be greater than this value.